### PR TITLE
Pass the argument `entity.table_ref()` instead of just `entity`.

### PR DESCRIPTION
**Pull request by josh-codes**
Pass the reference instead of the entity itself.

### CODE REVIEW

**Comment by billy1624** [comment]
Hey @josh-codes, I have added some commits, please check

**Comment by josh-codes** [comment]
Will test (:

File: Cargo.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ futures-util = { version = "^0.3" }
 log = { version = "^0.4", optional = true }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.3.1", path = "sea-orm-macros", optional = true }
-sea-query = { version = "^0.18.0", git = "https://github.com/SeaQL/sea-query.git", features = ["thread-safe"] }
+sea-query = { version = "^0.18.0", git = "https://github.com/josh-codes/sea-query.git", branch = "sea-query-change-type-into-table-ref", features = ["thread-safe"] }
```
Comment (tyt2y3) [inline Cargo.toml:33]: @billy1624 can help undo this line?

Comment (billy1624) [reply Cargo.toml:33]: Sure

**Review by billy1624 — APPROVED** [review]
Thanks!! @josh-codes

**Review by tyt2y3 — APPROVED** [review]

**Comment by josh-codes** [comment]
@tyt2y3 why was this merged? it is using the beta 2021 of rust version and had an outdated version string. [0.3.2](https://lib.rs/crates/sea-orm)

**Comment by tyt2y3** [comment]
> @tyt2y3 why was this merged? it is using the beta 2021 version of rust version and had an outdated version string. [0.3.2](https://lib.rs/crates/sea-orm)

I don't get it. Is it specific to this PR?

**Comment by billy1624** [comment]
I'm confused

**Comment by tyt2y3** [comment]
The final `changeset` does not touch the `Cargo.toml`

**Comment by billy1624** [comment]
> @tyt2y3 why was this merged? it is using the beta 2021 version of rust version and had an outdated version string. [0.3.2](https://lib.rs/crates/sea-orm)

I think you undo the changes already

**Comment by josh-codes** [comment]
Screenshot 2021-11-17 at 20 45 49
>
also, if it doesn't get compile with those settings, shouldn't we just use the correct settings?

**Comment by josh-codes** [comment]
> > @tyt2y3 why was this merged? it is using the beta 2021 version of rust version and had an outdated version string. [0.3.2](https://lib.rs/crates/sea-orm)
> 
> I think you undo the changes already

I did, but I wanted to ask first.

**Comment by billy1624** [comment]
Ahhhh I got what you mean

**Comment by billy1624** [comment]
You mean why is this still `0.3.1`. Correct?

https://github.com/SeaQL/sea-orm/blob/0deedddf2809da0fa81e9a54b24bea198410ad4d/Cargo.toml#L4-L14

**Comment by billy1624** [comment]
No worries, we will soon be releasing `0.4.0`

**Comment by josh-codes** [comment]
> No worries, we will soon be releasing `0.4.0`

Yes, and also, why is it version 2021, that's a beta version of rust.

**Comment by billy1624** [comment]
> > No worries, we will soon be releasing `0.4.0`
> 
> Yes, and also, why is it version 2021, that's a beta version of rust.

I think Rust Edition 2021 is stable now.
https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html#whats-in-1560-stable

**Pull request merged**